docs/07_ml_classification.md

```diff
@@ -1192,18 +1192,24 @@ multi_lasso_rs
 #> # Tuning results
 #> # 10-fold cross-validation 
 #> # A tibble: 10 × 5
-#>    splits               id     .metrics          .notes           .predictions  
-#>    <list>               <chr>  <list>            <list>           <list>        
-#>  1 <split [79119/8791]> Fold01 <tibble [40 × 5]> <tibble [1 × 1]> <tibble [175,…
-#>  2 <split [79119/8791]> Fold02 <tibble [40 × 5]> <tibble [0 × 1]> <tibble [175,…
-#>  3 <split [79119/8791]> Fold03 <tibble [40 × 5]> <tibble [0 × 1]> <tibble [175,…
-#>  4 <split [79119/8791]> Fold04 <tibble [40 × 5]> <tibble [0 × 1]> <tibble [175,…
-#>  5 <split [79119/8791]> Fold05 <tibble [40 × 5]> <tibble [0 × 1]> <tibble [175,…
-#>  6 <split [79119/8791]> Fold06 <tibble [40 × 5]> <tibble [0 × 1]> <tibble [175,…
-#>  7 <split [79119/8791]> Fold07 <tibble [40 × 5]> <tibble [0 × 1]> <tibble [175,…
-#>  8 <split [79119/8791]> Fold08 <tibble [40 × 5]> <tibble [0 × 1]> <tibble [175,…
-#>  9 <split [79119/8791]> Fold09 <tibble [40 × 5]> <tibble [0 × 1]> <tibble [175,…
-#> 10 <split [79119/8791]> Fold10 <tibble [40 × 5]> <tibble [1 × 1]> <tibble [175,…
+#>    splits               id     .metrics          .notes           .predictions
+#>    <list>               <chr>  <list>            <list>           <list>      
+#>  1 <split [79119/8791]> Fold01 <tibble [40 × 5]> <tibble [0 × 3]> <tibble>    
+#>  2 <split [79119/8791]> Fold02 <tibble [40 × 5]> <tibble [0 × 3]> <tibble>    
+#>  3 <split [79119/8791]> Fold03 <tibble [40 × 5]> <tibble [1 × 3]> <tibble>    
+#>  4 <split [79119/8791]> Fold04 <tibble [40 × 5]> <tibble [1 × 3]> <tibble>    
+#>  5 <split [79119/8791]> Fold05 <tibble [40 × 5]> <tibble [0 × 3]> <tibble>    
+#>  6 <split [79119/8791]> Fold06 <tibble [40 × 5]> <tibble [1 × 3]> <tibble>    
+#>  7 <split [79119/8791]> Fold07 <tibble [40 × 5]> <tibble [0 × 3]> <tibble>    
+#>  8 <split [79119/8791]> Fold08 <tibble [40 × 5]> <tibble [0 × 3]> <tibble>    
+#>  9 <split [79119/8791]> Fold09 <tibble [40 × 5]> <tibble [1 × 3]> <tibble>    
+#> 10 <split [79119/8791]> Fold10 <tibble [40 × 5]> <tibble [0 × 3]> <tibble>    
+#> 
+#> There were issues with some computations:
+#> 
+#>   - Warning(s) x1: from glmnet C++ code (error code -100); Convergence for 100th lam...   - Warning(s) x1: from glmnet C++ code (error code -100); Convergence for 100th lam...   - Warning(s) x1: from glmnet C++ code (error code -100); Convergence for 100th lam...   - Warning(s) x1: from glmnet C++ code (error code -100); Convergence for 100th lam...
+#> 
+#> Use `collect_notes(object)` for more information.
```

Comment:
We probably don't want these to get printed in the rendered site.

Reply:
removed!

Reply:
I still see these in the `.md`. Are we waiting on a re-render?

Reply:
Yes! Should be finished later today

Reply:
Okay I looked into this. That printing is part of the print method of `multi_lasso_rs`. So unless we do something hacky it will have to stay. We could not print that object all-together

Reply:
Probably not printing that object is a good option for now. We should note in #168 any changes we make so we can revert them if we solve this problem.

Reply:
Done!